dbt/models/intermediate/int_section_mapping.sql

```diff
@@ -36,13 +36,26 @@ sections as (
     from {{ ref('stg_dashboard__sections') }}
 ),
 
+section_instructors as (
+    select 
+        instructor_id,
```

Comment:
```suggestion
        instructor_id as teacher_id,
```
We should keep consistent labels, either user_id or teacher_id